.azure-pipelines/ultimate-pipeline.yml

```
@@ -317,9 +312,66 @@ stages:
         command: "Clean BuildProfilerHome"
 
     - publish: $(System.DefaultWorkingDirectory)/profiler/_build
-      displayName: Uploading linux profiler binaries
+      displayName: Uploading linux profiler output build folder
       artifact: linux-profiler-binaries-$(baseImage)
 
+    - publish: $(profilerHome)
+      displayName: Uploading linux profiler home artifact
+      artifact: linux-profiler-home-$(baseImage)
+
+- stage: package_linux
+  condition: and(succeeded(), eq(variables['isScheduledBuild'], 'False'))
+  dependsOn: [master_commit_id, build_linux_tracer, build_linux_profiler]
+  variables:
+    masterCommitId: $[ stageDependencies.master_commit_id.fetch.outputs['set_sha.master']]
+  jobs:
+  - template: steps/update-github-status-jobs.yml
+    parameters:
+      jobs: [package]
+
+  - job: package
+    dependsOn: []
```

We added default timeouts in #2768, so we should add it to all new stages

```suggestion
  - job: package
    dependsOn: []
    timeoutInMinutes: 60 #default value
```